src/dispatch/incident_priority/service.py

```diff
@@ -26,6 +33,22 @@ def get_default(*, db_session, project_id: int):
     )
 
 
+def get_default_or_raise(*, db_session, project_id: int) -> IncidentPriority:
+    """Returns the default incident_priority or raise a ValidationError if one doesn't exist."""
+    incident_priority = get_default(db_session=db_session, project_id=project_id)
+
+    if not incident_priority:
+        raise ValidationError(
+            [
+                ErrorWrapper(
+                    NotFoundError(msg="No default incident_priority defined."),
```

Comment:
We may want to make these messages a little bit more human readable. In this case, replacing `_` with a space.